src/Viewer.js

```diff
@@ -20,24 +20,56 @@ export default class Viewer extends Component {
     switch (e.code) {
       case "ArrowLeft":
       case "KeyA":
-        return this.prev();
+        if (!e.ctrlKey && !e.altKey && !e.shiftKey && !e.metaKey) {
+          this.prev();
+        }
+        return;
       case "ArrowRight":
       case "KeyD":
-        return this.next();
+        if (!e.ctrlKey && !e.altKey && !e.shiftKey && !e.metaKey) {
+          this.next();
+        }
+        return;
+      case "KeyF":
+        if (!e.ctrlKey && !e.altKey && !e.shiftKey && !e.metaKey) {
+          const video = document.querySelector("video");
+          if (video) {
+            (
+              video.requestFullscreen ||
+              video.webkitRequestFullScreen ||
+              video.mozRequestFullScreen ||
+              video.msRequestFullScreen ||
+              video.webkitEnterFullScreen ||
+              (() => null)
+            ).bind(video)();
+          }
+        }
+        return;
+      case "Space":
+        if (!e.ctrlKey && !e.altKey && !e.shiftKey && !e.metaKey) {
+          const video = document.querySelector("video");
+          if (video) {
+            if (video.paused) {
+              video.play();
+            } else {
+              video.pause();
+            }
+          }
+        }
       default:
         return;
     }
   };
 
   next = () => {
     this.setState(({ pos }, { links: q }) => ({
-      pos: Math.min(pos + 1, q.length - 1)
+      pos: Math.min(pos + 1, q.length - 1),
     }));
   };
 
   prev = () => {
     this.setState(({ pos }) => ({
-      pos: Math.max(pos - 1, 0)
+      pos: Math.max(pos - 1, 0),
```

Comment:
I've been running `npm start` since I started contributing, and honestly I never saw files getting autoformatted (unless I pressed my configured editor shortcut). But anyway, I've restored the comma related changes for now.

Reply:
huh, that's weird, sorry for assuming that you weren't already doing so. My intention for enforcing this is so that the only things that get checked in are meaningful semantic changes and not stylistic ones that don't affect functionality